meta-resin-common/recipes-support/resin-persistent-logs/resin-persistent-logs/resin-persistent-logs

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+set -e
+
+. /usr/sbin/resin-vars
+
+if [ "$PERSISTENT_LOGGING" == "true" ] && [ ! -d /var/log/journal ]; then
+    systemctl start var-log-journal.mount
+    journalctl --flush
+    echo "resin-persistent-logs: Persistent logging activated."
+elif [ "$PERSISTENT_LOGGING" == "true" ] && [ -d /var/log/journal ]; then
+    echo "resin-persistent-logs: Persistent logging already activated."
+    journalclt --flush
+elif [ "$PERSISTENT_LOGGING" == "false" ] && [ -d /var/log/journal ]; then
+    echo "resin-persistent-logs: Persistent logging was deactivated but system reboot is needed."
```

Comment:
It can but it breaks supervisor and openvpn. As well given that journald is not intended to be used to switch back to volatile I don't think it's safe at this point for use to try to hack it. We can improve it later when we feel more confident about it.